torch_glow/src/FuseKnownPatterns.cpp

```diff
@@ -337,6 +326,7 @@ void fuseKnownPatterns(std::shared_ptr<torch::jit::Graph> &graph) {
   std::call_once(onceFlag, []() {
     registerDummyOperator("glow::unpacked_quantized_linear");
     registerDummyOperator("glow::unpacked_quantized_conv2d");
+    registerDummyOperator("glow::FusedStack");
```

Comment:
Inconsistent naming

Reply:
Yeah, I was trying to name it like prim::FusedConcat but I'll change it